test/e2e/middleware-trailing-slash/test/index.test.ts

```diff
@@ -114,15 +114,17 @@ describe('Middleware Runtime trailing slash', () => {
         }
         delete middlewareWithoutEnvs.env
         expect(middlewareWithoutEnvs).toEqual({
-          files: expect.arrayContaining([
-            'server/edge-runtime-webpack.js',
-            'server/middleware.js',
-          ]),
+          files: process.env.TURBOPACK
+            ? expect.toBeArray()
```

Comment:
is this now too dynamic?

Reply:
Yeah: https://github.com/vercel/next.js/pull/69602#discussion_r1742383593

Reply:
Or better said: the webpack one is not dynamic enough 😄